graphql/match/mutations.ts

```diff
@@ -24,6 +26,10 @@ export class MutateMatchResolver {
 
         await createMatch(pupil, student, pool as ConcreteMatchPool);
 
+        await prisma.pupil_tutoring_interest_confirmation_request.updateMany({
```

Comment:
Can't you just call https://github.com/corona-school/backend/blob/master/common/match/interest.ts#L91 here?